index.html

```diff
@@ -1,378 +1,233 @@
-      <div class="img-scrolly-splash bg-img-aerial" id="splash">
-        
-        <div class="row bg-none" style="position: absolute; bottom: 0px; left: 0px; width: 100%;">
-          
-          <div class="col-xl-5 bg-none">
-            <img src="frontend/landing-page/img/logo-main.png" class="rounded mx-auto d-block" alt="Project Logo" id="logo-splash-project">
-          </div>
-
-          <div class="col-xl-2 bg-none"></div>
-
-          <div class="col-xl-5 bg-none"></div>
-          
-          <div class="col-xl-5 bg-none">
-            <h1 id="title-splash" class="centered">Wales Community Response<br>
-            Ymateb Cymunedol Cymru
-            </h1>
-          </div>
-
-          <div class="col-xl-2 bg-none">
-            <a href="#subhead" class="text-center"><div class="icons-down-splash col-hov-emphasis"><i class="fas fa-angle-down"></i> <i class="fas fa-chevron-down"></i> <i class="fas fa-angle-down"></i></div></a>
-          </div> 
-          
-          <div class="col-xl-5 bg-none"></div>
-
-        </div>
-
-      </div>
-    </div><!-- / row
-
-
-    <!-- Navbar -->
-    <nav class="navbar navbar-expand-md navbar-light bg-light font-head">
-      <div class="navbar-collapse collapse w-100 order-1 order-md-0 dual-collapse2">
-          <ul class="navbar-nav mr-auto">
-              <li class="nav-item active">
-              </li>
-          </ul>
-      </div>
-      <div class="mx-auto order-0">
-          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target=".dual-collapse2">
-              <span class="navbar-toggler-icon"></span>
-          </button>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="Mapping the Community Response to COVID-19 in Wales">
+    <meta name="author" content="The Dynamic Genetics Lab">
+    <meta name="generator" content="Jekyll v4.0.1">
+    <title>COVID-19 Community Response</title>
+    <link rel="shortcut icon" type="image/jpg" href="frontend/landing-page/img/logo-main-fav.png"/>
+
+    <!-- Bootstrap core CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+
+    <!-- Site specific CSS -->
+    <link rel="stylesheet" href="frontend/landing-page/css/variation.css">
+
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;400;500;600;800&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+      <!-- <img src="frontend/landing-page/img/logo-main.png" alt="Project Logo" id="logo-nav"> -->
+      <a class="navbar-brand" href="./index.html">COVID-19 Response Map</a>
```

Comment:
Just to point out that the `title` and the title on the navigation bar don't match :)
This is also replicated in the `user_guide` page.

Maybe we want to have the same title here?

Reply:
This is to do with the issue that Al raised about ITS not wanting us to change the <title>. The navigation bar is closer to the URL, so I think we should probably do it anyway and they should fix their monitoring instead. I'll leave it for now, but let's talk about it later.